sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamingRelation.scala

```diff
@@ -64,8 +64,20 @@ case class StreamingRelationExec(sourceName: String, output: Seq[Attribute]) ext
   }
 }
 
-object StreamingExecutionRelation {
-  def apply(source: Source): StreamingExecutionRelation = {
-    StreamingExecutionRelation(source, source.schema.toAttributes)
+case class StreamingRelationWrapper(child: LogicalPlan) extends UnaryNode {
+  override def isStreaming: Boolean = true
+  override def output: Seq[Attribute] = child.output
+}
+
```

Comment:
Add a new `StreamingRelationWrapper` relation to wrap the internal relation in each trigger. It keeps the `isStreaming` property.